### ISSUE TITLE
Convenient way of specifying python version

### ISSUE THREAD

**Issue by zsimic** [python-packaging/honesty#33]
Would be very convenient if a short form for `--python-version` was available, like `-p`, and if `major.minor` was accepted, such that we could run:

```
honesty deps -p3.7 requests
```

**Comment by thatch** [comment]
Yeah, with the caveat that it will pick an arbitrary patch version (probably 0), this is easily possible.  0.3.0b1 also defaults to the current python version, because that seemed better than defaulting to 3.7 :)
